pkg/cni/chained.go

```diff
@@ -64,22 +64,23 @@ func getCniConfigPath() (string, error) {
 		sort.Strings(files)
 		var confList *libcni.NetworkConfigList = nil
 
-		for _, confFile = range files {
-			confList, err = libcni.ConfListFromFile(confFile)
+		for _, file := range files {
+			confList, err = libcni.ConfListFromFile(file)
 			if err != nil {
-				err = fmt.Errorf("failed to read conflist: %v, %v", confFile, err)
+				err = fmt.Errorf("failed to read conflist: %v, %v", file, err)
 				log.Info(err)
```

Comment:
```suggestion
				log.Info("failed to read conflist: %v, %v", file, err)
```
Why not use one line of code to output the error log?

Reply:
Tep, this is not a good way to process err.

Let's handle errors in a separate pr